### DISCUSSION TITLE
A.D.D. ON!

### DISCUSSION THREAD

**Discussion by jasonjgardner** [jasonjgardner/jg-rtx#63] [Announcements]
![Screenshot of new sunflower painting](https://user-images.githubusercontent.com/1903667/157769952-404df49d-fb95-45ab-8b9e-cb85a9d19e85.png)

### Dev Blog
A few distractions made this release come a little later and lighter than planned. I spent too much time on my [RAINBOW! add-on update](https://github.com/jasonjgardner/minecraft-rtx-rainbow) and [JG RTX add-on](https://github.com/jasonjgardner/jg-rtx-addon). But hey, they're both very cool and you should check out the progress. 👍

It's also very hard to devote free time to Minecraft while being concerned about the invasion of Ukraine. Truthfully, I only found motivation after realizing I can use this release to attract thousands of generous people to this message:

> I've never accepted money for JG RTX. I've only asked for charitable donations ([and RTX ON keycaps](https://www.instagram.com/p/Ca5OR6zPJsN/)) instead of asking you to click links or buy me coffee. I didn't mind if you considered that a pass for a free download. **But now I do.** If you like JG RTX enough to donate and can afford to give _ANY_ amount of money, please give something to [Doctors Without Borders](https://donate.doctorswithoutborders.org/secure/donate), [UNICEF](https://www.unicefusa.org/) and other organizations helping Ukrainian refugees. If you can't afford to donate, you can always afford to advocate for peace. ☮️
> — Jason

### Changelog

**Added**
- Frog lights
- Cocoa
- Trip wire
- Potted warped and crimson roots
- Minecart
- Trapped chest
- Ender chest
- Pink coral plants
- Kelp
- Soul campfire animation and logs
- Campfire smoke
- Loom bottom

**Changed**
- Campfire animation
- Paintings
  - Replaced public domain paintings with a few new, original works
- Yellow coral plants
- Fire coral fan
- Loom
  - Changed wood material
- Default fog
  - Enabled thin, uniform fog
- Taiga fog
  - Fixed crazy-thick taiga fog
- Savanna fog

**Full Changelog**: https://github.com/jasonjgardner/jg-rtx/compare/v0.23.1...v0.24.1

>![Screenshot of JG RTX add-on blocks](https://user-images.githubusercontent.com/1903667/157771038-b6513237-06e5-4847-90aa-56917b51f6e0.png)
> Creeper petroglyph and stonebrick pillars from JG RTX add-on

<hr /><em>This discussion was created from the release <a href='https://github.com/jasonjgardner/jg-rtx/releases/tag/v0.24.1'>A.D.D. ON!</a>.</em>

**Comment by jasonjgardner** [comment]
P.S. https://www.doctorswithoutborders.org/streaming

**Comment by NumbThw** [comment]
Where is the addon download?

**Comment by w0203j** [comment]
Keep up the good work ... it looks sick as!
